app/models.py

```diff
@@ -0,0 +1,30 @@
+from marshmallow import Schema, fields
+import random
+
+class User():
+
+    def __init__(self, username, department, email, password):
+        self.user_id = random.randint(1,100)
```

Comment:
It would be better if you implemented this in a way that it increments by itself e.g by checking the amount of current users and adding 1.